docs/configuration/data_onboarding/email_o365.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Microsoft Office 365 Email
+permalink: /configuration/data_onboarding/email/email_o365/
+nav_order: 1
+parent: Email
+grand_parent: Data Onboarding
+---
+
+## **Microsoft Office 365 Email Data**
+
+The Splunk Add-on for Microsoft Office 365 Reporting Web Service is required to collect message trace data from Microsoft Office 365. 
+
+Splunkbase Download:
+[https://splunkbase.splunk.com/app/3720](https://splunkbase.splunk.com/app/3720) 
```

Comment:
@mahirchavda - New Office 365 Add-on itself let's us collect the email data. Need to check. Maybe for later, not for now.